count.rb

```diff
@@ -0,0 +1,40 @@
+class WordCount
+  attr_accessor :speech
+  attr_reader :counted_words
+
+  def load_file(path)
+    @speech = File.read(path)
+  end
+
+  def parse_words
+    @speech.gsub!(/[^0-9a-zA-Z\s]/, '')
+    @speech.gsub!(/\s+/, ' ')
+    @speech.downcase!.strip!
+    @speech = @speech.split(' ')
```

Comment:
You can use this split with your regular expression to split by invalid chars instead of by singe space. That will save you the substitution steps above.